FUNDING.yml

```
@@ -0,0 +1,4 @@
+# These are supported funding model platforms
+
+github: vanilla-os
+liberapay: fabricators
```

```suggestion
liberapay: fabricators

```
LGTM (Just this minor suggestion to fix eol), Thanks for your contribution. cc @pietrodicaprio